### DISCUSSION TITLE
Drop up option

### DISCUSSION THREAD

**Discussion by archonic** [saadeghi/daisyui#1709] [Ideas / feature requests]
Bootstrap and Flowbite have dropdowns which allow the button to stay in a bottom aligned position while content or a menu appear above it.
https://getbootstrap.com/docs/5.3/components/dropdowns/#dropup
https://flowbite.com/docs/components/dropdowns/#placement

I get that collapses can fill the roll of dropdowns, but I do need the ability to expand upwards. I would have expected drop up behaviour by just reversing the positions of the button and the content, but that doesn't happen.

**Comment by saadeghi** [comment]
Isn't this what you want? https://daisyui.com/components/dropdown/#dropdown-top

**Reply by archonic** [reply]
Yes -_- thank you haha
